eng/Subsets.props

```diff
@@ -326,6 +328,31 @@
       Test="true" Category="clr" Condition="'$(DotNetBuildFromSource)' != 'true' and '$(NativeAotSupported)' == 'true'"/>
   </ItemGroup>
 
+  <ItemGroup Condition="$(_subset.Contains('+tools.linker+'))">
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\linker\Mono.Linker.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\ILLink.Tasks\ILLink.Tasks.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\analyzer\analyzer.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\ILLink.RoslynAnalyzer\ILLink.RoslynAnalyzer.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\linker\ref\Mono.Linker.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\tlens\tlens.csproj" Category="tools" />
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\src\ILLink.CodeFix\ILLink.CodeFixProvider.csproj" Category="tools" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(_subset.Contains('+tools.linkertests+'))">
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\Mono.Linker.Tests\Mono.Linker.Tests.csproj"
+      Test="true" Category="tools" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\Mono.Linker.Tests.Cases\Mono.Linker.Tests.Cases.csproj"
+      Test="true" Category="tools" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\Mono.Linker.Tests.Cases.Expectations\Mono.Linker.Tests.Cases.Expectations.csproj"
+      Test="true" Category="tools" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\ILLink.Tasks.Tests\ILLink.Tasks.Tests.csproj"
+      Test="true" Category="tools" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\ILLink.RoslynAnalyzer.Tests\ILLink.RoslynAnalyzer.Tests.csproj"
+      Test="true" Category="tools" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
+    <ProjectToBuild Include="$(ToolsProjectRoot)linker\test\ILLink.RoslynAnalyzer.Tests.Generator\ILLink.RoslynAnalyzer.Tests.Generator.csproj"
+      Test="true" Category="tools" Condition="'$(DotNetBuildFromSource)' != 'true'"/>
+  </ItemGroup>
```

Comment:
I don't think that you need the `DotNetBuildFromSource` condition here as that subset won't be built automatically. Similar to the libs.tests subset which isn't built automatically either.

Reply:
For now, I think we don't want the burden of everyone building linker even if they won't use it and therefore I'm not including it in the DefaultSubsets. I will remove the conditions for DotNetBuildFromSource